packages/react-accordion/package.json

```diff
@@ -20,7 +20,7 @@
     "start": "storybook",
     "test": "jest",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
-    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/packages/react-accordion/src && yarn docs",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/react-accordion/src && yarn docs",
```

Comment:
@Hotell Do we plan to have a giant nearly-duplicated command like this in every package long-term? At least IMO, this PR is a good demonstration of how that's not particularly maintainable.

Reply:
- everything is very easy maintainable with our migrations/generators :)
- this is a temporary solution ([discussed here](https://github.com/microsoft/fluentui/pull/17449)) to be able to verify api-extractor output locally without building everything with lage